about.md

```diff
@@ -4,10 +4,15 @@ title: About
 permalink: /about/
 navigation_order: 6
 ---
-
 Monthly Music Hackathon NYC is a monthly all-day event for musicians and engineers to create new music-related projects from scratch, develop them throughout ~10 hours, then perform or present them in a concert in the evening.
 
+***
```

Comment:
Ah, thanks! I thought that I had to load the gem and give constants for it for it to work. 

I've removed it. The Table of contents isn't set up exactly as above, but really close, and works.